app/views/users/show.html.erb

```diff
@@ -1,2 +1,9 @@
 <h1>User details page </h2>
 <p>Details of the user which includes both Bio and Posts </h2>
+<h2><%= @user.Name%></h2>
```

Comment:
- [ ] As I said above :up: Please make sure that by clicking on each user , each user show page is looks like  the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) as required for this project  :+1:

Reply:
resolve the issue